02.calendar/cal.rb

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env ruby
+
+require 'optparse'
+require 'date'
+
+opt = OptionParser.new
+option = {}
+opt.on('-y [VAL]',Integer) {|v| option[:y] = v }
+opt.on('-m [VAL]',Integer) {|v| option[:m] = v }
+opt.parse!(ARGV)
+
+month = nil
+year = nil
+year = option[:y] || Date.today.year
+month = option[:m] || Date.today.month
+
+date = Date.new(year,month)
+
+first_day = Date.new(date.year, date.month, 1)
+last_day = Date.new(date.year, date.month, -1)
+puts "      #{date.year}年 #{date.month}月"
+
+day_of_week = ['日 月 火 水 木 金 土']
+puts day_of_week
+
+# 初日の位置を決める処理（日曜日であればスペースは空けない）
+print '   ' *first_day.wday
```

Comment:
そのうちLinter (Rubocop)に機械的に指摘されることではありますが、基本的に演算子の前後にはスペースを空けましょう〜
```suggestion
print '   ' * first_day.wday
```

Reply:
かしこまりました！